src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
 body {
+  padding: 0;
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__circle {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 50%;
+    background-color: #f6a603;
+    z-index: 1;
+  }
+
+  &__min-hand {
+    position: absolute;
+    bottom: 50%;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    transform-origin: bottom;
+    animation: move-min-hand 3600s steps(60) infinite;
```

Comment:
The `animation` shorthand property should be written in the order of `name duration timing-function delay iteration-count direction fill-mode`. Please ensure the order is correct and that all necessary values are included.